### DISCUSSION TITLE
DAI 2023-2024 - Practical work 1 - Simon Guggisberg and Jeremiah Steiner

### DISCUSSION THREAD

**Discussion by GuggisbergSimon** [heig-vd-dai-course/heig-vd-dai-course#85] [Show and tell]
We will create a CLI to convert images from webp to png or jpg and vice versa.
Webp is a file format that has yet to be supported everywhere despite its advantages. Having a tool to allow a quick and easy conversion from one format to the other two more popular ones are what we would like to achieve.

Students :
@BetterBlood
@GuggisbergSimon

Edit: Repository can be found here https://github.com/BetterBlood/DAI-PW1

**Comment by ludelafo** [comment]
Great! Looking forward to your project!

**Comment by GuggisbergSimon** [comment]
Submission of the work based on the latest commit on our branch main, hash is : f89f84a2e2193fee47dc9f85f44e78613ffa261c
https://github.com/BetterBlood/DAI-PW1/commit/f89f84a2e2193fee47dc9f85f44e78613ffa261c
